nixos/mixins/telegraf.nix

```diff
@@ -129,6 +129,7 @@ in
           device = [ "rpc_pipefs" "lxcfs" "nsfs" "borgfs" ];
         };
         diskio = { };
+        internal = { };
```

Comment:
Do you got some interesting alert rules that I could add here as well?

Reply:
I was thinking that these could be displayed on grafana but we could write rules as well.